stable/database/templates/_helpers.tpl

```diff
@@ -512,13 +512,33 @@ release: {{ .Release.Name | quote }}
 {{- end }}
 {{- end -}}
 
+{{/*
+Renders the storage group labels. IMPORTANT: Adding new entries must be done
+with caution as these labels are rendered in the PVC spec which is immutable.
+The storage group name can not be reconfigured by definition.
+*/}}
+{{- define "database.storageGroupLabels" -}}
+{{- if .Values.database.sm.storageGroup -}}
+{{- if .Values.database.sm.storageGroup.enabled -}}
+storage-group: {{ include "database.storageGroup.name" . | quote }}
```

Comment:
We follow the convention of prefixing the label and annotation keys with `cp.nuodb.com/` in the Control Plane. Should we adopt something similar with the Helm charts?

Reply:
Actually, "nuodb.com/" prefix is used only for KAA annotations. All other labels/annotations rendered by NuoDB Helm charts don't follow this concept. I would leave it like this for now.